doc/tutorials/vuart_configuration.rst

```diff
@@ -214,3 +214,131 @@ started, as shown in the diagram below:
+* When legacy vuart[1] is available, it is vUART1. PCI-vUART can't be vUART1 unless vuart[1] is not set.
+
+Setup ACRN vUART Using Configuration Tools
+==========================================
+
+When user setup ACRN VM configurations with PCI-vUART, it is better to use configuration tools. Because PCI resources are required: BDF number, address and size of mmio registers, address and size of MSIX entry tableas. They can't conflict with another PCI devices.
+Futhermore, whether PCI-vUART can use vuart_idx=0 and vuart_idx=1, depend on legacy vUART settings.
+So, configuration tools will override your settings in `How to configure a console port <https://projectacrn.github.io/latest/tutorials/vuart_configuration.html#how-to-configure-a-console-port>`_ and `How to configure a communication port <https://projectacrn.github.io/latest/tutorials/vuart_configuration.html#how-to-configure-a-communication-port>`_
+User can configure Legacy vUART and PCI-vUART both in ``./misc/vm_configs/xmls/config-xmls/<board>/<scenario>.xml``. For example, if VM0 has a legacy vUART0 and a PCI-vUART1, VM1 has no legacy vUART but has a PCI vUART0 and a PCI vUART1, VM0-vUART1 and VM1-vUART1 are connected to each other. User should configure like this:
+
+.. code-block:: none
+
+   <vm id="0">
+      <legacy_vuart id="0">
+        <type>VUART_LEGACY_PIO</type>     /* vuart[0] is console port */
+        <base>COM1_BASE</base>            /* vuart[0] is used */
+        <irq>COM1_IRQ</irq>
+      </legacy_vuart>
+      <legacy_vuart id="1">
+        <type>VUART_LEGACY_PIO</type>
+        <base>INVALID_COM_BASE</base>     /* vuart[1] is not used */
+      </legacy_vuart>
+      <console_vuart id="0">
+        <base>INVALID_PCI_BASE</base>     /* PCI-vUART0 can't be used, because vuart[0] */
+      </console_vuart>
+      <communication_vuart id="1">
+        <base>PCI_VUART</base>            /* PCI-vUART1 is communication port, connect to vUART1 of VM1 */
+        <target_vm_id>1</target_vm_id>
+        <target_uart_id>1</target_uart_id>
+      </communication_vuart>
+   </vm>
+
+   <vm id="1">
+      <legacy_vuart id="0">
+        <type>VUART_LEGACY_PIO</type>
+        <base>INVALID_COM_BASE</base>     /* vuart[0] is not used */
+      </legacy_vuart>
+      <legacy_vuart id="1">
+        <type>VUART_LEGACY_PIO</type>
+        <base>INVALID_COM_BASE</base>     /* vuart[1] is not used */
+      </legacy_vuart>
+      <console_vuart id="0">
+        <base>PCI_VUART</base>            /* PCI-vUART0 is console port */
+      </console_vuart>
+      <communication_vuart id="1">
+        <base>PCI_VUART</base>            /* PCI-vUART1 is commincation port, connect to vUART1 of VM0 */
+        <target_vm_id>0</target_vm_id>
+        <target_uart_id>1</target_uart_id>
+      </communication_vuart>
+   </vm>
+
+The ACRN vUART related xml fields:
+ - 'id' in <legacy_vuart>, value of vuart_idx, id=0 is for legacy vuart[0] configuration, id=1 is for vuart[1].
```

Comment:
```suggestion
 - ``id`` in <legacy_vuart>, value of vuart_idx, id=0 is for legacy vuart[0] configuration, id=1 is for vuart[1].
```

Use these double-backticks for all fields below please.

Reply:
OK, will use 2 backticks